otter/test/convergence/test_gathering.py

```diff
@@ -381,9 +405,19 @@ def nodes_req(lb_id, nodes):
 
 
 def node_feed_req(lb_id, node_id, response):
-    return lb_req(
-        'loadbalancers/{}/nodes/{}.atom'.format(lb_id, node_id),
-        False, response)
+    if isinstance(response, Exception):
```

Comment:
It looks like the type of the parameters got more important here - can you add a docstring?